BusBoard.Api/IncomingBusesApi.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using Newtonsoft.Json.Linq;
+
+namespace BusBoard.Api
+{
+    public class IncomingBusesApi
+    {
+        private const string PrimaryKey = "1056f6a94b6e44b48368f54a155429a3";

```

Comment:
I'd avoid storing secrets in plaintext in repository. You can use something like [this](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows)

Reply:
Although this is beyond the scope of the exercise, but it's more like an FYI